types/remarkable/package.json

```diff
@@ -1,7 +1,6 @@
 {
     "private": true,
     "dependencies": {
-        "highlight.js": "^9.7.0",
-        "@types/highlight.js": "^9.7.0"
+        "highlight.js": ">=10"
```

Comment:
highlight.js v9 produces the following warning on install:

```
-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*

  Verion 9 of Highlight.js has reached EOL.  It will no longer
  be supported or receive security updates in the future.
  Please upgrade to version 10 or encourage your indirect
  dependencies to do so.

  For more info:
  
  https://github.com/highlightjs/highlight.js/issues/2877
  https://github.com/highlightjs/highlight.js/blob/master/VERSION_10_UPGRADE.md
 
-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
```

v10 also ships with Typescript types out of the box.

Reply:
I would normally expect this to match the underlying library, which appears to depend on `"^9.7.0"`.  Incidentally, that seems to be a devdependency, so it's not immediately obvious why it would be required by the types?

Reply:
You are right. I just wanted to get rid of the warning and the newer types are only shipped with highlight.js itself. I reverted the commit.